src/Coordination/KeeperStorage.cpp

```diff
@@ -752,6 +808,21 @@ struct KeeperStorageCreateRequestProcessor final : public KeeperStorageRequestPr
             path_created += seq_num_str.str();
         }
 
+        if (path_created.starts_with(keeper_system_path))
+        {
+            auto error_msg = fmt::format("Trying to create a node inside the internal Keeper path ({}) which is not allowed. Path: {}", keeper_system_path, path_created);
+
+            if (keeper_context.server_state == KeeperContext::Phase::INIT && !keeper_context.ignore_system_path_on_startup)
+                throw Exception(
+                    ErrorCodes::LOGICAL_ERROR,
+                    "{}. Ignoring it can lead to data loss. "
+                    "If you still want to ignore it, you can set 'keeper_server.ignore_system_path_on_startup' to true.",
+                    error_msg);
+
+            LOG_ERROR(&Poco::Logger::get("KeeperStorage"), fmt::runtime(error_msg));
+            return {KeeperStorage::Delta{zxid, Coordination::Error::ZBADARGUMENTS}};
```

Comment:
Consider making common function for this

Reply:
Maybe even move it to `KeeperStorage::preprocessRequest` (but I'm not sure if `Request::getPath()` will work for all kinds of requests)

Reply:
I'll just create for `throw/LOG` helper function so the messages can be specific.